### PR TITLE
Creating Portfolio

### PR DESCRIPTION
**Pull request by SydneyCaulfeild**
I have removed the original index.html file and added a main.html file to act as the home page for my portfolio. I have modified the style.css file to style this page. I have also added three subpage html files with simple templates.

### CODE REVIEW

**Comment by SydneyCaulfeild** [comment]
I have also completed the background page with a text description and image gallery.

File: portfolio/src/main/webapp/background.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    <body>
+      
+      <div class="header">
+        <h1>My Background</h1>
+        <p>This page is all about my background.</p>
+      </div>
+      
+      <!-- Element where the heading's navigation bar will be inserted. -->
+      <header></header>
```
Comment (lipczak-g) [inline portfolio/src/main/webapp/background.html:19]: I'm curious, would \<header /\> be sufficient?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,249 @@
...
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+ 
+ /**************************** styling for the index.html file (main page) *************************/
+.header {
+  padding: 80px;
+  text-align: center;
+  background: #1abcbc;
```
Comment (lipczak-g) [inline portfolio/src/main/webapp/style.css:11]: FYI, Google standard is to sort properties alphabetically

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,249 @@
...
+  clear: both;
+  display: table;
 }
+ 
+/* Add a background color on hover */
+.btn-group button:hover {
+  background-color: #158b8b;
+}
+
+ /************************* Style for the image gallery in background.html **************************/
```
Comment (lipczak-g) [inline portfolio/src/main/webapp/style.css:106]: As we discussed typically these would be broken down into separate files.

**Review by lipczak-g — APPROVED** [review]

File: portfolio/src/main/webapp/background.html (hunk 1 above)
Comment (SydneyCaulfeild) [reply portfolio/src/main/webapp/background.html:19]: I don't think so. I tried making this change but everything after the <header /> tag wouldn't load.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (SydneyCaulfeild) [reply portfolio/src/main/webapp/style.css:11]: Ok thank you! I will go through later and update this.

**Pull request merged**